packages/theme-default/src/components/Link.svelte

```diff
@@ -9,7 +9,7 @@
   export let highlight = true
   export let withBase = true
 
-  $: isExternal = /^https?/.test(to)
+  $: isExternal = /^(https?)|(mailto:)/.test(to)
```

Comment:
It might be a good idea to look for `//:`, as there are many protocols that could be used and no internal path should have that substring (although it is possible on Linux: `/some/path//:whatever`, but `realpath` qould merge multiple slashes to a single slash).

```suggestion
  $: isExternal = /\/\/:/.test(to)
```